controllers/alias.go

```diff
@@ -242,3 +243,35 @@ func (r *ClusterClassReconciler) SetupWithManager(ctx context.Context, mgr ctrl.
 func (r *ClusterClassReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	return r.internalReconciler.Reconcile(ctx, req)
 }
+
+// ClusterResourceSetReconciler reconciles a ClusterResourceSet object.
+type ClusterResourceSetReconciler struct {
+	Client       client.Client
+	ClusterCache clustercache.ClusterCache
+
+	// WatchFilterValue is the label value used to filter events prior to reconciliation.
+	WatchFilterValue string
+}
+
+func (r *ClusterResourceSetReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, options controller.Options, partialSecretCache cache.Cache) error {
+	return (&ClusterResourceSetReconciler{
```

Comment:
These also have to use ClusterResourceSetReconciler from the other package (infinite recursion atm)

Same below for the other one